database/ydb/README.md

```diff
@@ -0,0 +1,30 @@
+# [YDB](https://ydb.tech/docs/)
+
+`ydb://[user:password@]host:port/database?QUERY_PARAMS`
+
+| URL Query  |               Description               |
+|:----------:|:---------------------------------------:|
+|   `user`   |         The user to sign in as.         |
+| `password` |          The user's password.           | 
+|   `host`   |         The host to connect to.         |
+|   `port`   |          The port to bind to.           |                                     |
+| `database` | The name of the database to connect to. |
+
+|   URL Query Params   |                         Description                         |
+|:--------------------:|:-----------------------------------------------------------:|
+|    `x-auth-token`    |                    Authentication token.                    |
+| `x-migrations-table` | Name of the migrations table (default `schema_migrations`). |
+|    `x-use-grpcs`     | Enables gRPCS protocol for YDB connections (default grpc).  |
```

Comment:
not required. Protocol defined in connection string directly

Reply:
golang-migrate uses scheme to determine db driver -- 
https://github.com/golang-migrate/migrate/blob/master/database/driver.go#L86